### ISSUE TITLE
Link Directories

### ISSUE THREAD

**Issue by undersquire** [undersquire/easymake#12]
Feature requested by @WittySmirk.
If you have any suggestions regarding link directories, please let me know via comment.

**Comment by MousieDev** [comment]
Yeah add "library_dirs" to build file and that would add -L options
